site/smart-accounts/session-keys/index.md

```diff
@@ -0,0 +1,63 @@
+---
+outline: deep
+head:
+  - - meta
+    - property: og:title
+      content: Introduction to using Session Keys
+  - - meta
+    - name: description
+      content: Learn about Alchemy's ERC-6900 Compatible Session Key Plugin.
+  - - meta
+    - property: og:description
+      content: Learn about Alchemy's ERC-6900 Compatible Session Key Plugin.
+  - - meta
+    - name: twitter:title
+      content: Introduction to using Session Keys
+  - - meta
+    - name: twitter:description
+      content: Learn about Alchemy's ERC-6900 Compatible Session Key Plugin.
+---
+
+# Session Key Plugin Overview
+
+The Session Key plugin lets your smart account add additional signers to your Modular Account with specific permissions.
+
+## Why Session Keys?
+
+### Skip duplicate confirmations
+
+Session keys unlock a simplified authentication process by allowing users to interact with apps without needing to confirm each action using their primary key. Instead, users create a session key with permissions specific to the app, then the app can use that key for future actions. This speeds up the user interaction and provides a smoother experience.
+
+### Automate actions
+
+With the ability to delegate specific permissions to session keys, users can automate actions within predefined limits. Session keys can be used to streamline processes like recurring payments, contract interactions, or any activity that benefits from automation.
+
+### Enhance security with permissions
+
+By using session keys, the exposure of the main private key is minimized. Even if a session key is compromised, the attacker would not gain access to the user's main account and funds. This layered approach to security helps in mitigating risks associated with key management and exposure.
+
+## Supported Permissions
+
+The session key plugin supports the following types of permissions for each key:
+
+### Time range
+
+Supports a start time and an end time for each key.
+
+### Access Control Lists
+
+Supports either an allowlist or a denylist for addresses. Optionally, access control lists may also specify specific functions on contracts to allow or deny.
+
+### ERC-20 Spending Limits
+
+Supports limiting how much of a specific ERC-20 token a key may spend. This may be a total for the key, or refreshing on an interval (e.g. 100 USDC per week).
```

Comment:
what about erc721, 1155, etc. other tokens? cc: @adam-alchemy

Reply:
I think right now it's limited to erc20 and native tokens

Reply:
We only support spending limits on ERC-20. For ERC-721 and ERC-1155, users can customize which methods a session key may call, but it doesn't support spending caps or refreshing spend limit intervals.